packages/terra-site/src/examples/form/examples/Input.jsx

```diff
@@ -1,16 +1,28 @@
-// remove eslint-disable once terra-form has been published
-/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import Input from 'terra-form/lib/Input';
-/* eslint-enable import/no-extraneous-dependencies */
 
-const InputExamples = () => (
-  <form>
-    <Input
-      name="foo"
-      defaultValue="bar"
-    />
-  </form>
-);
+class InputExamples extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { input: 'bar' };
+    this.handleChange = this.handleChange.bind(this);
+  }
 
+  handleChange(event) {
+    this.setState({ input: event.target.value });
+  }
+
+  render() {
+    return (
+      <div>
+        <form>
+          <Input name="foo" defaultValue="bar" onChange={this.handleChange} />
```

Comment:
Many of these examples are uncontrolled forms in a controlled state component, it might make sense to make these examples controlled forms instead of defining the state separately from the input value